### PR TITLE
feat(devx): Split and Edit full node guide

### LINKED ISSUE #4109: Improve iota-full-node guide
Please split the iota-full-node guide doc into docker and source instructions, and simplify the information flow to avoid redirecting the user away from the document

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* Split the Full Node guide into an overview, docker, source, and monitoring and pruning.
* Updated the relevant links.

## Notes

Is there any reason why we'd ask the users to fork our repo? It seemed like a lot of extra steps. 

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/4109

## Type of change

- Documentation Fix

## How the change has been tested

Docs were built locally.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that new and existing unit tests pass locally with my changes

## Release notes

Updated docs on running a full node.

### CODE REVIEW

File: docker/fullnode/README.md [hunk 1]
```diff
@@ -1,98 +1,134 @@
...
-https://docs.iota.org/operator/iota-full-node#set-up-from-source. and proceed with step 3.
+#### Devnet
 
-## Start your Iota Full node
+```yaml
+p2p-config:
+    seed-peers:
+    - address: /dns/access-0.r.testnet.iota.cafe/udp/8084
+        peer-id: 46064108d0b689ed89d1f44153e532bb101ce8f8ca3a3d01ab991d4dea122cfc
+    - address: /dns/access-1.r.testnet.iota.cafe/udp/8084
```
Comment (jkrvivian) [inline docker/fullnode/README.md]: Testnet peers in Devnet section. :eyes:

File: docker/fullnode/README.md [hunk 2]
```diff
@@ -1,98 +1,134 @@
...
 
-```shell
-docker compose up
+```yaml
+p2p-config:
+    seed-peers:
+    - address: /dns/access-0.r.devnet.iota.cafe/udp/8084
+        peer-id: 01589ac910a5993f80fbc34a6e0c8b2041ddc5526a951c838df3037e11ab0188
+    - address: /dns/access-1.r.devnet.iota.cafe/udp/8084
+        peer-id: 32875c547ea3b44fa08a711646cedb70fa0c97959d236578131505da09723add
```
Comment (jkrvivian) [inline docker/fullnode/README.md]: Devnet peers in Testnet section.

File: docker/fullnode/README.md [hunk 3]
```diff
@@ -1,98 +1,134 @@
...
+p2p-config:
+    seed-peers:
+    - address: /dns/access-0.r.testnet.iota.cafe/udp/8084
+        peer-id: 46064108d0b689ed89d1f44153e532bb101ce8f8ca3a3d01ab991d4dea122cfc
+    - address: /dns/access-1.r.testnet.iota.cafe/udp/8084
+        peer-id: 8ffd25fa4e86c30c3f8da7092695e8a103462d7a213b815d77d6da7f0a2a52f5
+```
 
-Run the following command to start the Iota fullnode in Docker:
+#### Tesnet
```
Comment (jkrvivian) [inline docker/fullnode/README.md]:
```suggestion
#### Testnet
```

File: docker/fullnode/README.md [hunk 4]
```diff
@@ -1,98 +1,134 @@
...
 ```
 
-### Stats (CPU/MEM USAGE %)
+## Monitoring
+
+### View Activity on Your Local Full Node with IOTA Explorer
 
-To view usage details for the Full node running in Docker, run the following command:
+The IOTA Explorer supports connecting to any network as long as it has `https` enabled. To view activity on your local
+Full Node, open the URL: [hhttps://explorer.rebased.iota.org/](https://explorer.rebased.iota.org/), and select Custom
```
Comment (jkrvivian) [inline docker/fullnode/README.md]:
```suggestion
Full Node, open the URL: [https://explorer.rebased.iota.org/](https://explorer.rebased.iota.org/), and select Custom
```

File: docs/content/operator/iota-full-node/docker.mdx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+description: Learn how to set up an IOTA Full node using docker.
```
Comment (jkrvivian) [inline docs/content/operator/iota-full-node/docker.mdx]:
```suggestion
description: Learn how to set up an IOTA Full Node using docker.
```

File: docs/content/operator/iota-full-node/overview.mdx [hunk 6]
```diff
@@ -0,0 +1,49 @@
+---
+title: Overview
+description: Operate an IOTA Full node to validate blockchain activities, like transactions, checkpoints, and epoch changes.
```
Comment (jkrvivian) [inline docs/content/operator/iota-full-node/overview.mdx]:
```suggestion
description: Operate an IOTA Full Node to validate blockchain activities, like transactions, checkpoints, and epoch changes.
```

**Review by jkrvivian — CHANGES_REQUESTED** [review]

File: docker/fullnode/README.md [hunk 7]
```diff
@@ -1,98 +1,134 @@
-# Use Docker to Run a Iota Full node Locally
+# Use Docker to Run an IOTA Full Node Locally
 
-Follow the steps in this Readme to install and configure a Iota Full node for testing locally using Docker. The instructions were validated on the following operating system/processor combinations:
+Use this guide to install and configure an IOTA Full Node locally using Docker for testing. These instructions are valid for the following operating system and processor combinations:
```
Comment (salaheldinsoliman) [inline docker/fullnode/README.md]:
```suggestion
Use this guide to install and configure an IOTA Full Node locally using Docker for testing. The instructions apply to the following operating system and processor combinations:
```

File: docker/fullnode/README.md [hunk 8]
```diff
@@ -1,98 +1,134 @@
...
+The genesis blob defines the IOTA network configuration. Before starting the Full Node, download the latest `genesis.blob` file to ensure compatibility with your version of IOTA.
 
-## Test the Iota Full node
+- [Testnet genesis blob](https://dbfiles.testnet.iota.cafe/genesis.blob):
+  `curl -fLJO https://dbfiles.testnet.iota.cafe/genesis.blob`
+- [Devnet genesis blob](https://dbfiles.devnet.iota.cafe/genesis.blob):
+  `curl -fLJO https://dbfiles.devnet.iota.cafe/genesis.blob`
+- [Devnet migration blob](https://dbfiles.devnet.iota.cafe/migration.blob):
+  `curl -fLJO https://dbfiles.devnet.iota.cafe/migration.blob`
+- [Mainnet genesis blob](https://github.com/iotaledger/iota/TODO):
```
Comment (salaheldinsoliman) [inline docker/fullnode/README.md]: Can we remove the Mainnet links and add a note that mainnet is not yet up, and a TODO as an invisible comment perhaps?

File: docs/content/_snippets/warning-advanced-instructions-node-setup.mdx [hunk 9]
```diff
@@ -0,0 +1,6 @@
+:::warning Advanced 
+
+These instructions are for advanced users. If you just need a local development environment,
```
Comment (salaheldinsoliman) [inline docs/content/_snippets/warning-advanced-instructions-node-setup.mdx]:
```suggestion
These instructions are for setting up a Full node for network participation. If you just need a local development environment, 
```

**Review by salaheldinsoliman — APPROVED** [review]
looks good, just a few comments

**Review by vivekjain23 — APPROVED** [review]
LGTM :)

**Review by jkrvivian — APPROVED** [review]

**Pull request merged**